.travis.yml

```diff
@@ -24,6 +24,9 @@ cache:
         - $HOME/.npm
 
 before_install:
+    - nvm install 0.12
```

Comment:
aware of that. priority was to fix the build first. not sure why we were using 0.10. 
will remove my changes.